packages/core/src/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable jsx-a11y/aria-props */
```

Comment:
Is this disabled because Radix handles the `aria` props?

Reply:
This is a remnant from when I had the props named `aria-title` and `aria-description` but I've since changed those prop names. I will remove this line